tests/unit/btree_stress_test.c

```diff
@@ -174,6 +172,7 @@ CTEST_TEARDOWN(btree_stress)
    clockcache_deinit(&data->cc);
    rc_allocator_deinit(&data->al);
    task_system_destroy(data->hid, &data->ts);
+   platform_heap_destroy(&data->hh);
```

Comment:
Missing in initial code; not an issue right now ... (Not sure why ASAN-builds are not tripping up ...)